test-d/non-empty-string.ts

```diff
@@ -0,0 +1,11 @@
+import {expectNever, expectType} from 'tsd';
+import type {NonEmptyString} from '../index';
+
+declare const empty: NonEmptyString<''>;
+expectNever(empty);
+
+declare const a: NonEmptyString<'a'>;
+expectType<NonEmptyString<'a'>>(a);
+
+declare const b: NonEmptyString<'b' | 'c'>;
+expectType<NonEmptyString<'b' | 'c'>>(b);
```

Comment:
The expected type and the actual type are identical, making these tests useless.

```suggestion
declare const a: NonEmptyString<'a'>;
expectType<'a'>(a);

declare const b: NonEmptyString<'b' | 'c'>;
expectType<'b' | 'c'>(b);
```